.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,44 @@
-## Description
+**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be compeleted as written. Provide known gaps to testing that may raise the risk of merging to production.
 
 
-## Original issue(s)
-department-of-veterans-affairs/va.gov-team#0000
+## Summary
 
+- *(Summarize the changes that have been made to the platform)*
+- *(If bug, how to reproduce)*
+- *(What is the solution, why is this the solution)*
+- *(Which team do you work for, does your team own the maintainence of this component?)*
+- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*
+
+## Related issue(s)
+
+- *Link to ticket created in va.gov-team repo*
+- *Link to previous change of the code/bug (if applicable)*
+- *Link to epic if not included in ticket*
 
 ## Testing done
 
+- *Describe what the old behavior was prior to the change*
+- *Describe the steps required to verify your changes are working as expected*
+- *Describe the tests completed and the results*
 
 ## Screenshots
```

Comment:
I think we should probably have different templates for the different repos (even if they're mostly the same), and I'd say the `Screenshots` section is proof of that. Makes sense for frontend, but not really for backend

Reply:
Yes we will have a slightly different template per repo. I however am not sure showing a screenshot of changes even for vets-api is out of the realm of possibility? Some of the issues we are seeing are that backend changes are being made without ensuring the frontend wasn't changed in a way that causes an issue. I would agree, most backend changes shouldnt have an impact on how the frontend loads things, but more than zero times per month we do.

Thoughts?

Reply:
For the backend PRs Im going to make screenshots optional